packages/ui/src/services/color/ColorCSSVariablesUpdater.stories.ts

```diff
@@ -1,12 +1,32 @@
-import { defineComponent } from 'vue'
-import ColorCSSVariablesUpdater from './ColorCSSVariablesUpdater.demo.vue'
+import { VaChip, VaDivider, VaButton } from '../../components'
+import { useColors } from '../../composables'
 
 export default {
-  title: 'ColorCSSVariablesUpdater',
-  component: ColorCSSVariablesUpdater,
+  title: 'services/ColorCSSVariablesUpdater',
 }
 
-export const Default = defineComponent({
-  components: { ColorCSSVariablesUpdater },
-  template: '<ColorCSSVariablesUpdater/>',
+export const Default = () => ({
+  components: { VaChip, VaDivider, VaButton },
+  methods: {
+    change (color) {
```

Comment:
Would be better to have it in script setup. Even though it doesn't produce any memory leaks or other issues it feels weird for me to use hooks in options api.